### ISSUE TITLE
Is it possible to bump versions and make git tags for the root package?

### ISSUE THREAD

**Issue by awxalbert** [changesets/changesets#980]
## Affected Packages

- @changesets/cli

## Problem

Currently I'm using changesets to manage a monorepo of applications on gitlab. When publishing multiple applications, changesets will make **git tags** for each of them. As we use [gitlab pipelines](https://docs.gitlab.com/ee/ci/pipelines/) to deploy applications, we will trigger a pipeline for each **git tag**, which is unnecessary in this case, because they are actually the same.

So in general, I think managing monorepo with applications have some difference from npm packages. When we want to cut a release, no matter how many apps changed, we will want them to be included in one release, thus, one version. In this use case, I come up with the idea that if there are any ways I could also bump versions of the root package.json and make git tags for it so I could achieve such goals, thanks a lot.

## Proposed solution

Bump versions and make git tags for the root package.

**Comment by reanimatedmanx** [comment]
Yep, please. Bumping for vis.

**Comment by jrnail23** [comment]
Same.
In my case, we have what I half-jokingly call a "pseudo-monorepo," where we have a React client app, a second (legacy) React client app, and the node.js server app that services these clients, each in their own workspace, under a root package.json.  The "pseudo" part is that while we use workspaces (npm v9) to organize the respective apps' dependencies, we build and deploy all of these apps together, regardless of which one may have changed.

We're not particularly interested in versioning the workspaces separately -- TBH, we've only ever bothered to version the root package.json.  We also would probably prefer to keep a single CHANGELOG.md for the entire repo.

Basically, I think our issues would be resolved if we could get Changesets to ignore the fact that we use workspaces at all, and just treat our root package.json as if it were a regular ol' single package repo.

Is there any way to do this?

**Comment by momentmaker** [comment]
Yes. This would be quite useful. Bump.

**Comment by ajgrande924** [comment]
bump

**Comment by rajat1saxena** [comment]
Bump

**Comment by smiccoli** [comment]
bump

**Comment by sebcomeau** [comment]
Any update @awxalbert?
